.github/workflows/check_build_push_template.yml

```diff
@@ -77,13 +77,31 @@ jobs:
         uses: actions/checkout@v4
       - name: Check that the image builds
         run: docker build -t rfswarm-image-for-check -f ${{ matrix.image }}/Dockerfile .
-      - name: Check that the built image runs
-        run: docker run -d --rm --name rfswarm-container-for-check rfswarm-image-for-check
-      - name: Sleep for 30 seconds
-        run: sleep 30s
-        shell: bash
-      - name: Shut docker container stop
-        run: docker stop rfswarm-container-for-check
+      - name: Build manager image
+        run: |
+          docker build -t rfswarm-docker-manager -f manager/Dockerfile .
+      - name: Create network
+        run: docker network create docker_network
+      - name: Run rfswarm agent
+        run: |
+          docker run --rm -d --name agent --hostname agent --network docker_network \
+          rfswarm-image-for-check
+      - name: Run rfswarm manager or pass
+        run: |
+          if ! [ -f $(pwd)/tests/${{ matrix.image }}/scenario.rfs ]; then
+            echo "Scenario file does not exist, nothing to run."
```

Comment:
OK. So, If I get it right if a scenario file is found in an image directory then , tests are run. I'm I right?

Reply:
yes, this means when testing base it runs the scenario for base, if it's testing selenium-firefox, it runs that scenario, I think this is a good solution :+1: